node/src/components/consensus/era_supervisor.rs

```diff
@@ -258,7 +258,7 @@ where
 
         let results = if should_activate {
             let secret = HighwaySecret::new(Rc::clone(&self.secret_signing_key), our_id);
-            highway.activate_validator(our_id, secret, timestamp)
+            highway.activate_validator(our_id, secret, timestamp.max(start_time))
```

Comment:
Shouldn't they be equal for the first era?

Reply:
Ah, no b/c `timestamp` might be *now* while `start_time` either in future or past.